kroma-validator/l2_output_submitter.go

```diff
@@ -199,6 +315,47 @@ func (l *L2OutputSubmitter) trySubmitL2Output(ctx context.Context) (time.Duratio
 	return 0, nil
 }
 
+// tryStartValidator checks if the validator can start and tries to start it.
+func (l *L2OutputSubmitter) tryStartValidator(ctx context.Context) (bool, error) {
+	cCtx, cCancel := context.WithTimeout(ctx, l.cfg.NetworkTimeout)
+	defer cCancel()
+	from := l.cfg.TxManager.From()
+	vaultStatus, err := l.valManagerContract.GetStatus(optsutils.NewSimpleCallOpts(cCtx), from)
+	if err != nil {
+		return false, fmt.Errorf("failed to fetch the vault status: %w", err)
+	}
+
+	if vaultStatus == val.StatusNone || vaultStatus == val.StatusInactive || vaultStatus == val.StatusActive {
+		l.log.Info("vault has not started yet", "currentStatus", vaultStatus)
+		return false, nil
+	}
+
+	if vaultStatus == val.StatusCanSubmitOutput {
+		l.log.Info("vault has started, no need to start again", "currentStatus", val.StatusStarted)
```

Comment:
```suggestion
	if vaultStatus >= val.StatusStarted {
		l.log.Warn("vault has already started, no need to start again", "currentStatus", val.StatusStarted)
```

Reply:
Would it be better to show warning in this case? This case means that the validator is in a right state to submit output, so  `log.Info` can be more suitable in this case. How do you think?

Reply:
Oh I see. `log.Info` is more appropriate.